packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -184,6 +184,11 @@ export class SDKClient {
     }
 
     async submitEthereumTransaction(transactionBuffer: Uint8Array): Promise<TransactionResponse> {
+        if (transactionBuffer.length > 4096) {
```

Comment:
I'm presuming an acceptance test case for a larger transaction will verify the support.

Reply:
The test is ready. Executed it locally with the updated version of the SDK and it passes, so once the [PR](https://github.com/hashgraph/hedera-sdk-js/pull/1182) is merged and released, we have to bump the SDK version in a relay and all should be good.